python/pyspark/sql/types.py

```diff
@@ -1249,7 +1249,7 @@ def _infer_schema_type(obj, dataType):
 }
 
 
-def _verify_type(obj, dataType, nullable=True):
+def _verify_type(obj, dataType, nullable=True, name="obj"):
```

Comment:
I guess this is only place that we print "obj" maybe? If so, let's set `name=None`.